wallet/file/sr25519_test.go

```diff
@@ -1,4 +1,4 @@
-package file
+package file //nolint: dupl
```

Comment:
Why is this needed?

Reply:
Linter complains that ed25519_test.go and sr25519_test.go are too similar although they aren't (different private key for the same mnemonic).